Source/WPEFramework/PluginServer.cpp

```diff
@@ -193,6 +194,8 @@ namespace PluginHost {
 
         WARNING_REPORTING_THREAD_SETCALLSIGN_GUARD(callsign.c_str());
     #endif
+      
+	    SetupCrashHandler();
```

Comment:
As already explained in https://github.com/rdkcentral/Thunder/pull/1592 there should be no need to check/install the crash handler each time a job is about to be run. Check the plugin for *not* modifying the signal handlers! 
This is to cover up an issue caused by a plugin. Fix the plugin.
It is also resource usage which should not be necessary. The handlers get installed at the beginning and that should be sufficient.
If that is not sufficient (if that is the issue) do it in the Initialize of the thread which only occures once, just before starting it.